packages/editor/src/editor.spec.ts

```diff
@@ -12,6 +12,29 @@ import { eventually } from '../../../testing/eventually';
 
 chai.use(sinonChai);
 
+const setupExternalEditor = async(base: string): Promise<string> => {
+  const tmpDoc = path.join(base, 'editor-script.js');
+
+  await fs.mkdir(path.dirname(tmpDoc), { recursive: true, mode: 0o700 });
+  await fs.writeFile(tmpDoc, '', { mode: 0o600 });
+
+  return tmpDoc;
+};
```

Comment:
Can we remove this function? It seems a bit like the only purpose it serves is to do `path.join(base, 'editor-script.js')` (and the `mkdir` call, but that can just go back before the `writeFile` call again)